hub/management/commands/import_new_constituencies.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaType
+
+
+class Command(BaseCommand):
+    help = "Import basic area information for new constituencies"
+
+    data_file = settings.BASE_DIR / "data" / "new_constituencies.csv"
```

Comment:
Would be nice to add a comment here, to remind us where this anonymous CSV came from. I’m pretty sure it was https://geoportal.statistics.gov.uk/datasets/ons::westminster-parliamentary-constituencies-future-boundaries-ew-bfc/about